src/input/__tests__/styled-components.test.js

```diff
@@ -9,7 +9,7 @@ import React from 'react';
 import {shallow} from 'enzyme';
 import {
   StyledRoot,
-  StyledInputEnhancer,
+  StyledStartEnhancer as StyledInputEnhancer,
```

Comment:
if we rename.. can we update the code below this line to `StyledStartEnhancer`? I find the unnecessary import naming hard to follow